src/components/project/project-rules/project.rules.ts

```diff
@@ -57,6 +59,7 @@ export interface EmailNotification {
   changedBy: Pick<User, 'id' | 'displayFirstName' | 'displayLastName'>;
   project: Pick<Project, 'id' | 'modifiedAt' | 'name' | 'step' | 'type'>;
   previousStep?: ProjectStep;
+  organizationName?: string | undefined;
```

Comment:
```suggestion
  primaryPartnerName?: string;
```
Would be better.
"Which partner?" - the primary one.
We'll try to stop using organizations probably, so favor that wording.